js/openfisca.js

```diff
@@ -108,24 +41,35 @@ function get(additionalParameters, callback) {
 		})
 }
 
-/** Updates the displayed values.
+/** API function
+
+Computes values based on the current main form state and the given additional parameters.
+*
+*@param	{Object}	[additionalParameters]	An object whose properties will be appended to the URL as query-string parameters.
+*@param	{Function<[XMLHttpRequest|SyntaxError], Object, Object>}	callback	A callback that will be called with three parameters: an optional error if something went wrong, the OpenFisca-computed values, and the full OpenFisca response if you want everything it sends back.
 */
-function update() {
-	const today = new Date(),
-		mm = ('0' + (today.getMonth() + 1)).slice(-2)
+function get(additionalParameters, callback) {
+	// Base url containing the list of desired output variables
+	var baseUrl = UI.getOutputVariables()
+	var input = UI.collectInput()
 
-	get({
-		contrat_de_travail_debut: today.getFullYear() + '-' + mm,
-	}, function(error, values, response) {
-		if (error) {
-			if (response && response.error)
-				return UI.showError(response.error)
+	Object.assign(input, additionalParameters) // merge parameters
 
-			UI.showError({ message: error })
-			throw error
-		}
+	return request(input, baseUrl, callback)
+}
+
+
+/** API backward compatibility function
+
+Creates an OpenFisca URL to the /formula endpoint, based on the current main form state and the given additional parameters.
+*
+*@param		{Object}	[additionalParameters]	An object whose properties will be appended to the URL as query-string parameters.
+*@returns	{String}	The URL for the OpenFisca query.
+*/
+const buildOpenFiscaQueryURL = (additionalParameters) => get(additionalParameters)
```

Comment:
No way it's used, this is private and has never been exposed as such. It has only been exposed as `buildURL`. Just export `get` as `buildURL` too.